src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/Categories/Blocks/form.html.twig

```diff
@@ -136,17 +136,20 @@
 
       {{ renderhook('displayBackOfficeCategory') }}
 
-      {% block category_tool_serp %}
-        <div class="form-group row">
-          <label class="form-control-label">{{ 'SEO preview'|trans({}, 'Admin.Global') }}</label>
-          <div class="col-sm">
-            <div id="serp-app" data-category-url="{{ categoryUrl }}"></div>
-            <small class="form-text">
-              {{ 'Here is a preview of how your page will appear in search engine results.'|trans({}, 'Admin.Global') }}
-            </small>
+      {# Do not display category preview while creating a category #}
+      {% if categoryUrl is not null %}
+        {% block category_tool_serp %}
+          <div class="form-group row">
+            <label class="form-control-label">{{ 'SEO preview'|trans({}, 'Admin.Global') }}</label>
+            <div class="col-sm">
+              <div id="serp-app" data-category-url="{{ categoryUrl }}"></div>
```

Comment:
I don't know if you checked but be careful since you remove the `#serp-app` you may have a JS error, because the Serp component expects this div to be present and I'm not sure if it handles its absence correctly (it should fail silently in this case)

Reply:
I did not check, but this code inside serp app check if the html node exists before running anything. 

```js
    if ($(selectors.container).length === 0) {
      return;
    }
```

Should I test the existence of the node before calling it or can we rely on it ?

Reply:
I guess the component does it by itself so it should be enough?

Reply:
No this is enough I just wanted to be sure If no errors pop out in the console it's already a good sign